payjoin-cli/Cargo.toml

```diff
@@ -2,7 +2,10 @@
 name = "payjoin-cli"
 version = "0.0.1"
 authors = ["Dan Gould <d@ngould.dev>", "Martin Habovstiak <martin.habovstiak@gmail.com>"]
-edition = "2018"
+repository = "https://github.com/payjoin/rust-payjoin"
+readme = "README.md"
```

Comment:
I believe [this is default behavior](https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field). Any reason to be explicit like it wasn't being picked up or incorrectly picking up payjoin/README.md?